### DISCUSSION TITLE
libp2p-crypto: marked as unimplementable/unsupported

### DISCUSSION THREAD

**Discussion by francoisthire** [libp2p/rust-libp2p#2965] [General]
Looking at here: https://libp2p.io/implementations/ , we can see that libp2p-crypto for the Rust implementation is marked as `unimplementable/unsupported`. However, there is no link to give more info about the reason why. Is it possible to get more data about what is unsupported or/and cannot be implemented?

**Comment by dariusc93** [comment]
I dont believe I ever seen a crate or any information or documentation regarding "libp2p-crypto" so maybe its missing or just been removed in the past? I honestly feel that page is a bit out dated considering some items on there are marked differently compared to what is implemented in rust-libp2p or maybe just marked different items without clarification (eg floodsub is marked as in progress/usable while it been done for awhile unless it is still a WIP, while with gossipsub, being implemented, havent done PX yet but still marked done. Unless im missing something, bootstrao is "missing" while kad has support for it, etc).

**Reply by francoisthire** [reply]
It would be nice to have this page up to date. This problem could be tracked by an issue maybe?

**Comment by thomaseizinger** [comment]
The source-code for the website is here: https://github.com/libp2p/website

Feel free to send a PR!

**Comment by mxinden** [comment]
Also maybe @DannyS03 has opinions here as you last updated this part of the documentation.
